platform-metrics/src/main/java/org/hypertrace/core/serviceframework/metrics/PlatformMetricsRegistry.java

```diff
@@ -380,8 +380,10 @@ public static DistributionSummary registerDistributionSummary(String name,
    * Registers metrics for GuavaCaches using micrometer's GuavaCacheMetrics under the given
    * cacheName for the given guavaCache
    */
-  public static <K, V> void registerCache(String cacheName, Cache<K, V> guavaCache) {
-    GuavaCacheMetrics.monitor(METER_REGISTRY, guavaCache, cacheName);
+
+  public static <K, V> void registerCache(String cacheName, Map<String,String> tags,Cache<K, V> guavaCache) {
```

Comment:
general pattern is to have tags (map) as the last parameter

Reply:
registerGauge seems to have this order of params. 
registerTimer, registerDistributionSummary too, though they have that pattern for internal functions. 
Nevertheless fixing it.